audio/out/ao_openal.c

```diff
@@ -175,15 +175,12 @@ static void uninit(struct ao *ao)
 
 static int init(struct ao *ao)
 {
-    MP_FATAL(ao, "broken.\n");
-    return -1;
-
     float position[3] = {0, 0, 0};
     float direction[6] = {0, 0, -1, 0, 1, 0};
     ALCdevice *dev = NULL;
     ALCcontext *ctx = NULL;
     ALCint freq = 0;
-    ALCint attribs[] = {ALC_FREQUENCY, ao->samplerate, 0, 0};
+    ALCint attribs[] = { ALC_FREQUENCY, ao->samplerate, 0, 0 };
```

Comment:
Unnecessary cosmetic change that also sort of goes against most of the coding style in this software.

Reply:
Done